crates/tuono/src/watch.rs

```diff
@@ -86,6 +98,13 @@ pub async fn watch() -> Result<()> {
 
     let build_ssr_bundle = build_react_ssr_src();
 
+    let env_files = fs::read_dir("./")
+        .unwrap()
```

Comment:
Looks like I left in an unwrap here :)

```suggestion
        .expect("Error reading env files from current directory")
```